samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,125 @@
+
+Overview
+********
+
+The sample demonstrates a device combining the CIS Central and BIS Source roles.
+As a CIS Central, it only receives data from CIS Peripheral.
+The received data is then broadcasted over a BIS.
+To provide the best performance and to minimize the number of conflicts between the CIS, BIS and ACL roles, the following measures are taken:
+* The same ISO Interval is used for both CIS and BIS. The value is set to 10 ms by default.
+* ACL Interval is set to an integer multiple of ISO Interval to avoid collisions. The value is set to 20 ms by default.
+* Periodic advertising of BIS is set to the same value as ACL Interval.
+* The number of retransmissions is set to 10, to ensure that packets are retransmitted in case of collisions.
+* The sample rejects incoming Connection Update requests from the peripheral to avoid changes to the ACL interval.
+* In addition, the following optional SDC-specific configurations are used:
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT` is set to 10 ms to ensure that there is enough time available to place the periodic advertising packet between ACL packets.
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT` is set to a small value 2.5 ms because the sample doesn't send any data over ACL connection (except Control procedures).
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_BIG_RESERVED_TIME_US` is set to half of the ISO Interval to ensure BIG parameters are selected in a way leaving enough time for ACL and CIG.
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_CIG_RESERVED_TIME_US` is set to half of the ISO Interval to ensure CIG parameters are selected in a way leaving enough time for ACL and BIG.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/iso_combined_bis_and_cis`
+
+.. include:: /includes/build_and_run_ns.txt
+
+
+Testing
+=======
+
+In addition to the device running this sample, two additional devices are required:
+* CIS Peripheral device that sends data over CIS.
+* BIS Sink device that receives data over BIS.
+Other sample applications can be used as CIS Peripheral and BIS Sink (for example, :ref:`bluetooth_isochronous_time_synchronization`)
+CIS Peripheral device should have the same name as the name set in :kconfig:option:`CONFIG_BT_DEVICE_NAME`.
+This name is used to connect to the correct device.
+
+After programming the sample to the development kit and programming other samples to CIS Peripheral device and BIS sink device, perform the following steps to test it:
+
+1. |connect_terminal_specific|
+#. Reset the kits.
+#. Wait until the sample connected to CIS Peripheral device and established CIS connection.
+#. Observe packets transmitted by CIS Peripheral to be retransmitted over BIS and received by BIS Sink.
+
+
+Sample output
+==============
+
+[00:00:00.454,589] <inf> bt_hci_core: HW Platform: Nordic Semiconductor (0x0002)
```

Comment:
This data should be in a `.. code-block::`

Also, you need to add a Dependencies section between this section and the References section.

Reply:
thanks, I added Dependencies and also added code-block section